src/main/java/com/azure/cosmos/examples/bulk/async/SampleBulkQuickStartAsync.java

```diff
@@ -0,0 +1,250 @@
+package com.azure.cosmos.examples.bulk.async;
```

Comment:
We should add the required license header. Please add the following on top of this file: 

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
will do